services/121-service/src/registration/services/registrations-pagination.service.ts

```diff
@@ -462,6 +488,24 @@ export class RegistrationsPaginationService {
         );
       case 'isNull':
         return queryBuilder.andWhere(`${uniqueJoinId}.${columnName} IS NULL`);
+      case 'moreThan':
+        return queryBuilder.andWhere(
+          `${uniqueJoinId}.${columnName}::numeric > :value${uniqueJoinId}`,
+          { [`value${uniqueJoinId}`]: value },
+        );
+      case 'lessThan':
+        return queryBuilder.andWhere(
+          `${uniqueJoinId}.${columnName}::numeric < :value${uniqueJoinId}`,
+          { [`value${uniqueJoinId}`]: value },
+        );
+      case 'between':
```

Comment:
Would it be possible for some value to "fall in between" these scenario's?

So for timestamps of `<date>T00:00:00:000Z` exactly?

So wouldn't it be necessary to have `>=`/`<=` instead of only `>`/`<`?

And/Or is the below `BETWEEN`-command/operator already including "the edges"?

Reply:
But this situation is still good right? The filter is supposed to work for greater than and not greater than or equal. Maybe we should call about this if we feel like we are talking past each other